src/components/CommentsList.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState,useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import getArticles from '../utils/api'
+import Comments from './Comments'
+
+function CommentsList() {
+
+    const [commentsList,setCommentsList] = useState([])
+
+    const [isLoading,setIsLoading] = useState(true)
+
+    const {id} = useParams()
+
+    useEffect(()=>{
+      getArticles(id + "/comments")
```

Comment:
we should have a different util function - get comments by id - therefore you don't need to modify the original string in the get articles function and we can have a better separation of concerns